src/LodeRunner.API/src/Core/Startup.cs

```diff
@@ -145,7 +138,18 @@ public static void ConfigureServices(IServiceCollection services)
         {
             AddSwaggerServices(services);
 
-            services.AddCors();
+            int portNumber = AppConfigurationHelper.GetLoadRunnerUIPort(SystemConstants.LodeRunnerUIDefaultPort);
```

Comment:
Question, before CORS was configured for DevEnvironment only, now it does not differentiate between Dev and Production, is that something we need to address?

Reply:
app.UseCors() is only executed when env.IsDevelopment() is true